plugins/infinispan80/src/main/java/org/radargun/service/InfinispanEmbeddedContinuousQuery.java

```diff
@@ -37,7 +36,7 @@ public void removeContinuousQuery(String cacheName, Object cqListener) {
         return service.getCache(cacheName);
     }
 
-    public static class IspnContinuousQueryListener implements org.infinispan.query.continuous.ContinuousQueryResultListener {
+    public static class IspnContinuousQueryListener implements org.infinispan.query.continuous.ContinuousQueryListener {
```

Comment:
Hmm which version did this API change take place in? If it was in 8.1.x, a new implementation should be created in separate plugin as per previous comment, wdyt?